test/errorMiddleware.test.js

```diff
@@ -0,0 +1,85 @@
+const listen = require('test-listen')
+const { createServer, stopServer, generateRequest } = require('./utils')
```

Comment:
I :heart: this test utils, I think they are invaluable for this kind of testing. And I just learnt about the `stopServer` function, I will check how it works in a minute :smile: . Thanks!